.github/workflows/pluto_build.yml

```diff
@@ -82,6 +82,13 @@ jobs:
           echo "is_minor: ${{ needs.version.outputs.is_minor_version }}"
           ../../bash/docker_container_setup.sh
 
+      - name: clean out shared libs
```

Comment:
yea this was in my mad dash to avoid running our disk space during a github action run. I got it from [this suggestion](https://github.com/actions/runner-images/issues/2840#issuecomment-790492173)

I don't know how much it helped because the build didn't pass until I added `VACUUM`. but it takes > 2 hours to build so I haven't tried it without this / tried logging disk space after major build steps

Reply:
weird, but if it works, 🤷‍♂️. Maybe you could add a comment in the build file linking to that issue. Or even just documenting it in the commit body